docs/guides/admin/docs/installation/java-version.md

```diff
@@ -0,0 +1,10 @@
+Supported Java Versions
```

Comment:
Should we list maximum jdk versions as well?

Reply:
The versions overlap and are the exact versions which are supported. For example, Opencast 16.x supports Java 11 and 17. But if you already missed that… Any idea to highlight that better?

Reply:
PEBKAC, I just needed to spend more time parsing what I was reading :D